src/ocean/util/serialize/contiguous/Serializer.d

```diff
@@ -290,7 +290,7 @@ struct Serializer
                         // Recurse into static array elements which contain a
                         // dynamic array.
 
-                        foreach (element; s.tupleof[i])
```

Comment:
Ah I see. I didn't get the connection between the use of `ref` and the change of `s.tupleof[i]` -> `field`. Maybe just add a bit of explanation in the commit message.